CPP14.md

```diff
@@ -35,6 +35,15 @@ auto identity = [](auto x) { return x; };
 int three = identity(3); // == 3
 std::string foo = identity("foo"); // == "foo"
 ```
+```c++
```

Comment:
I think the existing snippet (above) already demonstrates type deduction pretty well.

Reply:
Removed extraneous example